.github/workflows/ci.yml

```diff
@@ -73,23 +109,11 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --all -- --check
-
-      - name: Run cargo clippy
-        if: matrix.toolchain == 'beta'
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          # As `return_self_not_must_use` became `pedantic` in the 1.60.0-nightly channel,
-          # we allow the warning to avoid unnecessary code fixes.
-          # See: https://github.com/rust-lang/rust-clippy/issues/8197
-          #
-          # TODO: Remove this branch once stable-1.59.0 is released.
-          args: --all-features --all -- -D warnings --allow clippy::return_self_not_must_use
+          args: --workspace -- --check
 
       - name: Run cargo clippy
         if: matrix.toolchain != 'beta'
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-features --all -- -D warnings
+          args: ${{ maxtix.features }} --workspace -- -D warnings
```

Comment:
```suggestion
          args: ${{ matrix.features }} --workspace -- -D warnings
```

Reply:
Do you know of a way to test the validity of these offline?

Reply:
Done.

Reply:
AIFAIK, there is no easy way to verify GitHub Actions offline😅